sdk/core/Azure.Core.Experimental/src/DynamicJson.cs

```diff
@@ -0,0 +1,588 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Dynamic;
+using System.IO;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+#pragma warning disable 1591
+
+namespace Azure.Core
+{
+    /// <summary>
+    ///
```

Comment:
Maybe `Represents a JSON value returned from or provided to service operations.  It can be used as either a dynamic object or via its accessors.` for now?

Reply:
Oh, you didn't do any of these yet...   :smile:  It's probably okay for Experimental but it'd be good to fix them up as soon as folks are happy with the broad shape of the API.